lib/modules/announcements.js

```diff
@@ -199,7 +197,8 @@ function isPostRelevant(post) {
 		return true;
 	}
 
-	if (postTitle.search(`${currentBrowser}`) > -1) {
+
+	if (postTitle.search(`${BrowserDetect.browser}`) > -1) {
```

Comment:
```
- if (postTitle.search(`${BrowserDetect.browser}`) {
+ if (postTitle.search(BrowserDetect.browser) {
```

If you're not adding any text or formatting to a variable, then simply use the variable.